Pod/Classes/LSClockState.h

```diff
@@ -1,15 +1,15 @@
 #import <Foundation/Foundation.h>
 NS_ASSUME_NONNULL_BEGIN
 
-/// A straight port/copy of the `rl-cruntime-common` ClockState javascript prototype.
-@interface LSClockState : NSObject
+@protocol LSTimeProvider
+- (NSDate *)currentTime;
+- (SInt64)offsetInMicroseconds;
```

Comment:
@bensigelman agreed - I originally implemented it just as a block so the client libraries don't have to implement interfaces. I've used interface just for more structured refactoring and compiler help. Will revisit to see if we can simplify even further